### PR TITLE
Converted from erb to slim and changed reload scores thing

### LINKED ISSUE #3: Move from ERB to Slim
Getting Started - http://slim-lang.com/
To Translate Current Pages - http://erb2slim.com/

### PR DESCRIPTION
**Pull request by chesterhow**
@moexmen/interns 

Fixes #3

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -37,6 +37,8 @@ group :development, :test do
   gem 'byebug'
 end
 
+gem 'slim-rails'
```
Comment (abeltay) [inline Gemfile]: Can you move this above line 35 for easier reading?

File: app/assets/javascripts/scores.js [hunk 2]
```diff
@@ -0,0 +1,11 @@
+// # Place all the behaviors and hooks related to the matching controller here.
+// # All this logic will automatically be available in application.js.
+// # You can use CoffeeScript in this file: http://coffeescript.org/
```
Comment (abeltay) [inline app/assets/javascripts/scores.js]: Don't need both `//` and `#`

**Comment by handshou** [comment]
:gem:

File: app/views/layouts/_menu_bar.html.slim [hunk 3]
```diff
@@ -0,0 +1,30 @@
+.menu.bottom-line
+  .v-center
+    ul
+      - if @active == 1
+        li.active
+          a[href="/players"]
```
Comment (abeltay) [inline app/views/layouts/_menu_bar.html.slim]: `= link_to players_path`?

**Pull request merged**